docs/platforms/javascript/common/configuration/integrations/console.mdx

```diff
@@ -11,8 +11,16 @@ supported:
   - javascript.gcp-functions
   - javascript.koa
   - javascript.electron
+  - javascript.nextjs
+  - javascript.sveltekit
+  - javascript.remix
+  - javascript.astro
 ---
 
+<Alert level="info">
+  This integration only works inside a Node.js environment.
```

Comment:
So it works for bun and deno as well, we should make that clear. What do you think about something like?

```suggestion
  This integration only works inside server environments (Node.js, Bun, Deno).
```